official/nlp/docs/train.md

```diff
@@ -159,7 +159,7 @@ python3 train.py \
 
 ### Fine-tuning Sentence Classification with BERT from TF-Hub
 
-<details>
+
```

Comment:
You can remove the 2 empty lines after removing the tag.

Reply:
Removed extra lines